presto-hive/src/main/java/io/prestosql/plugin/hive/s3select/PrestoS3ClientFactory.java

```diff
@@ -95,6 +96,7 @@ private AmazonS3 createS3Client(Configuration config)
         Duration connectTimeout = Duration.valueOf(config.get(S3_CONNECT_TIMEOUT, defaults.getS3ConnectTimeout().toString()));
         Duration socketTimeout = Duration.valueOf(config.get(S3_SOCKET_TIMEOUT, defaults.getS3SocketTimeout().toString()));
         int maxConnections = config.getInt(S3_SELECT_PUSHDOWN_MAX_CONNECTIONS, defaultMaxConnections);
+        boolean s3PathStyleAccess = config.getBoolean(S3_PATH_STYLE_ACCESS, defaults.isS3PathStyleAccess());
```

Comment:
Can you make this a separate commit, since it's a bug fix, unrelated to the CSV comments change? Title it something like

> Respect path-style-access config for S3Select

Reply:
Yes, it should be a separate commit